### PR TITLE
Second batch of agreed risk scores added

### PR DESCRIPTION
**Pull request by SeanSuckling**
Second batch of risk scores for business type added - scores as agreed at team workshop.

### CODE REVIEW

**Review by Nathan-Philippo — CHANGES_REQUESTED** [review]
Sean, we seem to have lost the names of quite a few of the business types, was this intentional? if not they will need to be re added before we can approve

**Comment by SeanSuckling** [comment]
I can't see any missing business types, can you give an example? Some of the formatting is out re the answers to the business type questions which might be giving the impression that business types are missing.

File: business-types.json
Comment (Nathan-Philippo) [inline business-types.json:285]: Sean - no business type on this one, can see the changes to the scores but i cannot determine the business type

**Comment by Nathan-Philippo** [comment]
Sean,

So line 186 to 194, there is no name business type, only the scores that you have changed.

I have tried leaving a comment of the got hub file, can you see it?

regards

Nathan Philippo
Regulating Our Future Programme
Food Standards Agency
02072768372
07775032157
www.food.gov.uk<http://www.food.gov.uk>

**Review by Nathan-Philippo — CHANGES_REQUESTED** [review]
76: remove the new H - **DONE**
190: why is depuration not give 20 for MOP - **20 ADDED** 
271: Is milk pasteurised, if not its fine - **LEAVE AS 20 - PRECUATIONARY PRINCIPLE**
378: does not handle food - **AGREE, CHANGED TO 5** 
407: does not handle food - **AGREE, CHANGED TO 5**
445: can i see definition - **MEETING ARRANGED TO REVIEW DEFINITIONS**
479: can i see definition - **MEETING ARRANGED TO REVIEW DEFINITIONS**
638: why handling score of 10? - **OPINION IS THAT MOST NEWSAGENTS/POST OFFICES NOW RETAIL PRE-PACKED HIGH RISK FOODS THAT REQUIRE TEMPERATURE CONTROL (EG SANDWICHES, DAIRY ETC)**
839: why only 5 for consumers at risk if fast for restaurant is a 10? - **FAST FOOD RESTAURANT (736) WRONG, CHANGED TO 5**
916: MOP cannot be a 5. - **SCORE FOR VULNERABLE GROUPS ENTERED ON WRONG LINE. 916 CORRECTED TO 0, LINE 917 CHANGED FROM 0 TO 5**

**Comment by SeanSuckling** [comment]
See comments above. I will commit the amendments.

**Review by Nathan-Philippo — APPROVED** [review]

**Comment by Nathan-Philippo** [comment]
Sean, approved but once we have discussed the definition of some business types we may need to revisit 445 and 479

**Comment by SeanSuckling** [comment]
Agree, thanks

Sean

**Pull request merged**
